### ISSUE TITLE
Menu icon doesn't allow user to enter menu with keyboard

### ISSUE THREAD

**Issue by acressall** [ParabolInc/parabol#5736]
When tabbing through the dashboard nav the menu icon can open and close the menu, but the user can't enter the menu. They must continue tabbing _past_ the dashboard to reach the menu.

Related WCAG success criteria:

- 2.4.7 Focus Visible
- 2.1.1 Keyboard

How serious is the bug? P4

- OS: All
- Browser: All

## Acceptance Criteria
When a user is focused on the sidebar icon they should be able to:
1. Open the sidebar (click, tap, enter), and have focus start on the first item inside the nav
2. Close the sidebar (click, tap, escape), and maintain focus in the icon

## Estimated effort
1 hour to triage
